.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,10 @@
 # FreeCodeCamp - Wiki Pull Request Template
 
-----
+--------------------------------------------------------------------------------
 
 PLEASE REMOVE THIS WHOLE TEMPLATE BEFORE YOU SUBMIT YOUR PULL REQUEST
```

Comment:
To center the text, this can be use but not essential

<p align="center">PLEASE REMOVE THIS WHOLE TEMPLATE BEFORE YOU SUBMIT YOUR PULL REQUEST</p>